analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -236,6 +236,22 @@ def process(self, events):
                 elif region == "4j2b":
                     region_filter = ak.sum(selected_jets.btagCSVV2 > B_TAG_THRESHOLD, axis=1) >= 2
                     selected_jets_region = selected_jets[region_filter]
+                    
+                    pt_var_modifier = (
+                        events[event_filters][region_filter][pt_var]
```

Comment:
```suggestion
                        events[pt_var][event_filters][region_filter]
```

I think this does the same and is more intuitive to read, at least for me: we attached `pt_var` directly to the `event` previously. This restores the consistency with the `else` version two lines below as well. Since the 4j1b version is the same, please change it there as well.